sdk/src/store.rs

```diff
@@ -1440,6 +1440,32 @@ impl Store {
         }
     }
 
+    /// Embed the claims store as jumbf into an asset using an CoseSign box generated remotely. Updates XMP with provenance record.
+    #[cfg(feature = "async_signer")]
+    pub async fn save_to_asset_remotely_signed(
```

Comment:
May want to change this to  save_to_asset_remote_signed to match  Manifest.embed_remote_signed.

Reply:
Sure